binding/SkiaSharp/GRContext.cs

```diff
@@ -63,6 +63,18 @@ public static GRContext CreateVulkan (GRVkBackendContext backendContext, GRConte
 			}
 		}
 
+		public static GRContext CreateDirect3D (GRD3dBackendcontext backendContext, GRContextOptions options)
```

Comment:
I hate myself so much... But this may need to be 3d with a lowercase d for consistency. So yuck to look at and I have no idea what I was thinking at the time. Sorry universe.